app/models/creator.rb

```diff
@@ -0,0 +1,115 @@
+class Creator < ActiveRecord::Base
+  has_many :content
+  has_and_belongs_to_many :users
+  validates :p_id, :provider, :username, presence: true
+
+  include HTTParty
+  LIMIT_PER_PAGE = 10
+
+  #method to configure twitter
+  def twit
+    Seemore::Application.config.twitter
+  end
+
+  def has_image?
+    if avatar_url.nil?
+      false
+    else
+      true
+    end
+  end
+
+#calls methods for each provider
+  def get_content
+    if provider == "twitter"
+      get_tweets
+    elsif provider == "vimeo"
+      get_videos
+    else
+      raise "Content provider provided is not recognized (vimeo, twitter)."
+    end
+    self.last_updated = DateTime.now
```

Comment:
Rails automatically keeps an "updated_at" column in the database, so you don't need to create/maintain your own last_updated column. Check out your schema file to see that :)